libraries/plugins/peerplays_sidechain/peerplays_sidechain_plugin.cpp

```diff
@@ -499,6 +500,42 @@ void peerplays_sidechain_plugin_impl::on_new_objects(const vector<object_id_type
                approve_proposal( son_id, proposal->id );
                continue;
             }
+
+            if(proposal->proposed_transaction.operations.size() == 1
+            && proposal->proposed_transaction.operations[0].which() == chain::operation::tag<chain::bitcoin_transaction_send_operation>::value) {
+               approve_proposal( son_id, proposal->id );
+               continue;
+            }
+         }
+      }
+
+      if( object_id.is<chain::bitcoin_transaction_object>() ) {
```

Comment:
All sidechain specific code should be placed in sidechin handler class. We need more abstraction here

Reply:
ok,  I see